### PR TITLE
JBIDE-27533: add microprofile support in product

### PR DESCRIPTION
**Pull request by sbouchet**
Signed-off-by: Stephane Bouchet <sbouchet@redhat.com>

### CODE REVIEW

**Comment by sbouchet** [comment]
testPR

File: features/com.jboss.devstudio.core.feature/feature.xml [hunk 1]
```diff
@@ -58,6 +58,7 @@
   <includes id="org.jboss.tools.ws.jaxrs.feature" version="0.0.0"/>
   <includes id="org.jboss.tools.ws.jaxws.feature" version="0.0.0"/>
   <includes id="org.jboss.tools.wtp.runtimes.tomcat.feature" version="0.0.0"/>
+  <includes id="org.jboss.tools.cdi.microprofile.feature" version="0.0.0"/>
```
Comment (jeffmaury) [inline features/com.jboss.devstudio.core.feature/feature.xml:61]: Move it next to other CDI features for better maintenance

**Review by jeffmaury — CHANGES_REQUESTED** [review]

File: features/com.jboss.devstudio.core.feature/feature.xml (hunk 1 above)
Comment (sbouchet) [reply features/com.jboss.devstudio.core.feature/feature.xml]: fixed.

**Review by jeffmaury — APPROVED** [review]

**Comment by sbouchet** [comment]
testPR

**Pull request merged**